v1/fx/fx.gunk

```diff
@@ -0,0 +1,207 @@
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package fx
+
+import (
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+// FX holds all the details about a foreign exchange object.
+type FX struct {
+	// BankID is an identifier for the bank on this fx transaction.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// FromCurrencyCode is the currency to transfer from.
+	FromCurrencyCode string `pb:"2" json:"from_currency_code"`
+
+	// ToCurrencyCode is the currency that we are transferring to.
+	ToCurrencyCode string `pb:"3" json:"to_currency_code"`
+
+	// Rate is the exchange rate of the foreign exchange.
+	Rate string `pb:"4" json:"rate"`
+
+	// InverseRate is the inverse of the exchange rate of the foreign exchange.
+	InverseRate string `pb:"5" json:"inverse_rate"`
+
+	// EffectiveDate is the effective date of the foreign exchange quote.
+	EffectiveDate string `pb:"6" json:"effective_date"`
+}
+
+// CreateFXRequest is a request envelope to create a foreign exchange quote.
+type CreateFXRequest struct {
+	// FX is the foreign exchange information to create.
+	FX FX `pb:"1" json:"fx"`
+}
+
+// UpdateFXRequest is a request envelope to update a foreign exchange quote.
+type UpdateFXRequest struct {
+	// FX is the foreign exchange information to update.
+	FX FX `pb:"1" json:"fx"`
+}
+
+// GetCurrentFXRateRequest is a request envelope to get a foreign exchange rate.
+type GetCurrentFXRateRequest struct {
+	// FromCurrencyCode is the currency to transfer from.
+	FromCurrencyCode string `pb:"1" json:"from_currency_code"`
+
+	// ToCurrencyCode is the currency that we are transferring to.
+	ToCurrencyCode string `pb:"2" json:"to_currency_code"`
+}
+
+// FXService provides branch-related operations.
+type FXService interface {
+	// GetCurrentFXRate retrieves the current foreign exchange rate of the two
+	// specified currency code.
+	//
+	// +gunk http.Match{
+	//         Method: "GET",
+	//         Path:   "/v1/fx/{FromCurrencyCode}/{ToCurrencyCode}",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"FX"},
+	//         Description: "Retrieve the exchange rate from a currency to another",
+	//         Summary:     "Retrieve exchange rate between two currencies",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/fxFX",
```

Comment:
i think this should be `"#/definitions/fxGetCurrentFCRateResonse",`

Reply:
the response type is `FX`, there is no `GetCurrentFCRateResonse` defined.